x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/form/index.tsx

```diff
@@ -65,17 +65,22 @@ export const EventFiltersForm: React.FC<EventFiltersFormProps> = memo(
 
     const handleOnBuilderChange = useCallback(
       (arg: ExceptionBuilder.OnChangeProps) => {
-        if (isEmpty(arg.exceptionItems)) return;
         dispatch({
           type: 'eventFiltersChangeForm',
           payload: {
-            entry: {
-              ...arg.exceptionItems[0],
-              name: exception?.name ?? '',
-              comments: exception?.comments ?? [],
-              os_types: exception?.os_types ?? [OperatingSystem.WINDOWS],
-            },
-            hasItemsError: arg.errorExists || !arg.exceptionItems[0].entries.length,
+            ...(arg.exceptionItems[0]
```

Comment:
Is this a validation that [0] is a truthy value or that it exists?

perhaps `arg.exceptionItems.length > 0` is more specific of what is being validated here? or `arg.exceptionItems[0] !== undefined`

Reply:
Yeah, totally agree, `arg.exceptionItems[0] !== undefined` looks good to me